cornedbeef/src/main/java/com/swiftkey/cornedbeef/BubbleCoachMark.java

```diff
@@ -135,9 +141,9 @@ protected void updateView(CoachMarkDimens<Integer> popupDimens, CoachMarkDimens<
         // Optional parameters with default values
         protected boolean showBelowAnchor = false;
         protected float target = 0.5f;
-        
-        public BubbleCoachMarkBuilder(Context context, View anchor, String message) {
-            super(context, anchor, message);
+
+        public BubbleCoachMarkBuilder(Context context, View anchor, String message, int textColor) {
```

Comment:
Does this need to be in the constructor? Could it be a setter like for the background color?